packages/cashier/src/Components/side-note.jsx

```diff
@@ -1,39 +1,82 @@
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import React from 'react';
 import { DesktopWrapper, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
 import 'Sass/side-note.scss';
 
-const SideNoteBullet = ({ children }) => (
+const SideNoteTitle = ({ title }) => (
+    <Text className='side-note__title' weight='bold' as='p'>
+        {title}
+    </Text>
+);
+
+const SideNoteText = ({ children }) => (
+    <Text className='side-note__text' size='xxs' as='p'>
+        {children}
+    </Text>
+);
+
+const SideNoteBullet = ({ children, component }) => (
     <div className='side-note__bullet-wrapper'>
         <div className='side-note__bullet' />
-        <span>{children}</span>
+        {component ? children : <SideNoteText>{children}</SideNoteText>}
     </div>
 );
 
-const SideNote = ({ notes, title, has_bullets }) => (
-    <div className='side-note'>
-        {title && (
-            <DesktopWrapper>
-                <Text className='side-note__text' weight='bold' as='p'>
-                    {title}
-                </Text>
-            </DesktopWrapper>
-        )}
-
-        {notes.map((note, i) =>
-            has_bullets ? (
-                <SideNoteBullet key={i}>{note}</SideNoteBullet>
-            ) : (
-                <Text key={i} className='side-note__text' size='xxs' as='p'>
-                    {note}
-                </Text>
-            )
-        )}
-    </div>
-);
+const SideNote = ({ side_notes, title, has_bullets = true }) => {
+    const side_note_title =
```

Comment:
Could you add this const in `SideNoteTitle`

Reply:
Fixed. I moved it to the `SideNoteTitle`.